etc/config/c++.amazon.properties

```diff
@@ -7,143 +7,147 @@ objdumper=/opt/compiler-explorer/gcc-8.1.0/bin/objdump
 # GCC for x86
 group.gcc86.compilers=g412:g447:g453:g464:g471:g472:g473:g474:g481:g482:g483:g484:g485:g490:g491:g492:g493:g494:g510:g520:g530:g540:g550:g6:g62:g63:g71:g72:g73:g81:gsnapshot
 group.gcc86.groupName=GCC x86-64
+group.gcc86.baseName=x86-64 gcc
```

Comment:
How I wish we could derive all this stuff from the compilers themselves :crying_cat_face: